WcaOnRails/app/inputs/date_picker_input.rb

```diff
@@ -56,7 +56,10 @@ def date_view_header_format
 
   def date_options_base
     {
-        locale: I18n.locale.to_s,
+        # FIXME deal with this correctly
+        # Right now passing another locale raises an error in 'moment',
+        # complaining the locale is not loaded.
+        locale: I18n.default_locale.to_s,
```

Comment:
To keep track of TODOs/FIXMEs, I like to have separate github issues. Can you make a new issue for this and link to it here?